### PR TITLE
Tests: assert changed to doAssert

### PR DESCRIPTION
**Pull request by jlp765**
Also added "file:" to test header

### CODE REVIEW

**Comment by jlp765** [comment]
This is for Issue #3630

**Comment by jlp765** [comment]
tests/modules/trecinca.nim
tests/modules/trecincb.nim

failed in Travis,  

but on my win7 PC,  these all pass
``koch test c modules``
``koch test r tests\modules\trecinca.nim``
``koch test r tests\modules\trecincb.nim``

So I am mystified as to what is wrong.

File: tests/assign/moverload_asgn2.nim [hunk 1]
```diff
@@ -1,3 +1,7 @@
+discard """
```
Comment (Araq) [inline tests/assign/moverload_asgn2.nim]: FYI discard sections for files not starting with 't' are not processed in any way.

**Comment by Araq** [comment]
> Also added "file:" to test header

Why? Seems completely redundant, the default is the file name.

**Comment by jlp765** [comment]
Oops.  Stuffed up that rebase   :-(

**Comment by jlp765** [comment]
I'll close this and do it again

**Pull request closed without merging**